src/pages/api/scrape.js

```
@@ -8,6 +9,202 @@ const cleanText = (text) => {
+            return item;
+          }
+        }
+      }
+      // Handle direct Recipe object
+      else if (jsonData['@type'] === 'Recipe' || 
+               (Array.isArray(jsonData['@type']) && jsonData['@type'].includes('Recipe'))) {
+        return jsonData;
+      }
+    } catch (e) {
+      console.error('Error parsing LD+JSON:', e);
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)